converters/alignment_to_stockholm.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# given an alignment file and alignment tool, call the correct stockholm parser
+
+if [ $# -lt 2 ]
+then
+  echo "$0: takes 2 arguments: alignment file, alignment tool" 1>&2
+  echo "alignment tool options: cross_match, RepeatMasker" 1>&2
+  exit 1
+fi
+
+alignment_file="$1"
+alignment_tool="$2"
+
+if [ "$alignment_tool" = "cross_match" ]; then
+  python cm_to_stockholm.py "$alignment_file"
+fi
+
+if [ "$alignment_tool" = "RepeatMasker" ]; then
+  python rm_to_stockholm.py "$alignment_file"
+fi
```

Comment:
We ought to account for this logic in the Python script instead of a shell script. Let's not worry about it here, but I suggest we create an issue for it.